extensions/typescript-language-features/extension-browser.webpack.config.js

```diff
@@ -60,30 +57,20 @@ module.exports = withBrowserDefaults({
 				}))
 			],
 		}),
-		// @ts-ignore
-		new CopyPlugin({
-			patterns: [
-				{
-					from: '../node_modules/typescript/lib/tsserver.js',
-					to: 'typescript/tsserver.web.js',
-					transform: async (content) => {
-						const dynamicImportCompatPath = path.join(__dirname, '..', 'node_modules', 'typescript', 'lib', 'dynamicImportCompat.js');
-						const prefix = fs.existsSync(dynamicImportCompatPath) ? fs.readFileSync(dynamicImportCompatPath) : undefined;
-						const output = await Terser.minify(content.toString());
```

Comment:
withBrowserDefaults doesn't minify because it sets `mode: none`, but there's a comment that says `mode: production` will minify. So I didn't include a minify entry in tsserver.web.js' config.